build/source/engine/coupled_em.f90

```diff
@@ -453,16 +453,18 @@ lightSnow            ! maximum interception capacity an inverse function of new
  if(err/=0)then; err=20; message=trim(message)//trim(cmessage); return; end if
 
  ! adjust canopy temperature to account for new snow
- call tempAdjust(&
-                 ! input: derived parameters
-                 canopyDepth,                 & ! intent(in): canopy depth (m)
-                 ! input/output: data structures
-                 mpar_data,                   & ! intent(in):    model parameters
-                 prog_data,                   & ! intent(inout): model prognostic variables for a local HRU
-                 diag_data,                   & ! intent(out):   model diagnostic variables for a local HRU
-                 ! output: error control
-                 err,cmessage)                  ! intent(out): error control
- if(err/=0)then; err=20; message=trim(message)//trim(cmessage); return; end if
+ if(computeVegFlux)then ! logical flag to compute vegetation fluxes (.false. if veg buried by snow)
+  call tempAdjust(&
+                  ! input: derived parameters
+                  canopyDepth,                 & ! intent(in): canopy depth (m)
+                  ! input/output: data structures
+                  mpar_data,                   & ! intent(in):    model parameters
+                  prog_data,                   & ! intent(inout): model prognostic variables for a local HRU
+                  diag_data,                   & ! intent(out):   model diagnostic variables for a local HRU
+                  ! output: error control
+                  err,cmessage)                  ! intent(out): error control
+  if(err/=0)then; err=20; message=trim(message)//trim(cmessage); return; end if
```

Comment:
Indent the `if(err/=0)` line

Reply:
Do we need to skip more than just this routine? E.g. should `canopySnow` also be skipped?